browser/src/Services/QuickOpen.ts

```diff
@@ -62,10 +67,12 @@ export class QuickOpen {
                         })
                 } else {
                     // TODO: This async call is being dropped, if we happen to use the promise
-                    return recursiveQ(process.cwd())
-                        .then((files: string[]) => {
-                            this._showMenuFromFiles(files)
-                        })
+                    return glob("**/*", {
```

Comment:
Later, for #269, we could use the `match` event to incrementally populate the list.